tensorboard/components/tf_tensorboard/test/tensorboardTests.ts

```diff
@@ -40,7 +62,9 @@ namespace tf_tensorboard {
         });
       });
 
-      describe('top right global icons', function() {
+      // TODO(psybuzz): Restore or remove these tests. This folder's tests seems
+      // to have not been running since the GitHub repo history was tracked.
```

Comment:
nit: instead of describing the history, for TODOs at least, describe precondition that can resolve this TODO.

Reply:
Rewritten to note that the test was not running, and at least 1 reason for their failure.